### ISSUE TITLE
Action keeps retrying on any and all errors

### ISSUE THREAD

**Issue by mrchief** [mrchief/aws-creds-okta-action#8]
```
botocore.exceptions.ConfigParseError: Unable to parse config file: /github/home/.aws/config
An error occurred while getting credentials. Attempt #7.
Waiting 30 seconds before retrying...
```

In this case, obtaining credentials were not an issue, so the action should have aborted. Instead it keeps retrying because of this generic [check](https://github.com/mrchief/aws-creds-okta-action/blob/master/entrypoint.sh#L22)

**Comment by CesarManriqueH** [comment]
I opened [PR-9](https://github.com/mrchief/aws-creds-okta-action/pull/9) to address this issue.

**Comment by mrchief** [comment]
Closed via #9
